yagb-cli/src/helper/boolean.ts

```diff
@@ -0,0 +1,53 @@
+export function boolval<T>(value: T): boolean;
```

Comment:
In the end, we only use the code path that return `0` or `1`. I would prefer if we had just a single short function that returns `1` on `1|on|true` , `0` on `0|off|false`, and `-1 ` for invalid cases. This would save some code and would also allow to avoid repeating these cases in the switch statements below.